src/System.IO.FileSystem/tests/DirectoryInfo/GetSetAttributes.cs

```diff
@@ -38,12 +38,39 @@ public void InvalidParameters()
             Assert.Throws<ArgumentException>(() => Set(string.Empty, FileAttributes.Normal));
         }
 
-        [Fact]
-        public void NonExistentFile()
+        // In NetFX we ignore "not found" errors, which leaves the attributes
+        // state as invalid (0xFFFFFFFF), which makes all flags true.
+
+        [Theory, MemberData(nameof(TrailingCharacters))]
+        public void GetAttributes_MissingFile(char trailingChar)
+        {
+            string path = GetTestFilePath();
+            DirectoryInfo info = new DirectoryInfo(path + trailingChar);
+            Assert.Equal((FileAttributes)(-1), info.Attributes);
```

Comment:
Nit: These new tests don't use the `Get`/`Set` methods that enable child classes to utilize the tests for other methods.

I don't know offhand if there actually are any other classes that inherit this so it may just be a style issue.